src/checkbox/Checkbox.ts

```diff
@@ -141,7 +141,7 @@ export default class Checkbox extends CheckboxBase<CheckboxProperties> {
 
 		if (label) {
 			checkboxWidget = w(Label, {
-				classes: this.classes(css.root, ...stateClasses),
+				extraClasses: this.classes(css.root, ...stateClasses),
```

Comment:
@msssk This usage of `extraClasses` isn't quite right. The idea is to pass an object that references the class name key you wish to add classes to and a space-separated list of the classes you want to add to it. The idea being that in most cases you'd only be adding one or two classes via `extraClasses`. I guess that label is a bit of an edge case as it's being used to wrap an input. Something like the following should get you where you need to be:

``` typescript
function getExtraClasses(classes: { [ string ]: boolean }): string {
    return Object.keys(classes).reduce((classNamesString: string, className) => {
        if (classes[className] === true) {
            classNamesString += ` ${className}`;
        }
        return classNamesString;
    }, '').trim();
}

if (label) {
	checkboxWidget = w(Label, {
		extraClasses: { 'root': getExtraClasses(this.classes(css.root, ...stateClasses).get()) },
		formId,
		label
	}, children);
}
else {
	checkboxWidget = v('div', {
		classes: this.classes(css.root, ...stateClasses)
	}, children);
}
```